README.md

```diff
@@ -1,34 +1,43 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+Welcome to our Github repository! This app is built with Next.js, Tailwind.css, pnpm, Prisma, and Supabase.
 
-## Getting Started
+# Running the app
 
-First, run the development server:
+## Testing database
 
-```bash
-npm run dev
-# or
-pnpm run dev
-```
+To run the app with testing database, follow these steps:
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+### Prerequisities
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+-   [Node.js](https://nodejs.org/en/)
+-   [pnpm](https://pnpm.io/)
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
+### Steps
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+1. Clone this repository to your local machine.
+1. Navigate to the root directory of the project and run `pnpm install` to install the dependencies.
+1. Set the environment variables in a `.env` file in the root directory. You can find a sample `.env.testing` file in the root directory.
```

Comment:
The word "set" evokes setting a file, I tried to find the nonexisting file. "Find a sample" sounds like it is s sample and it is necessary to change something.